Utilities/Hooks/prepare-commit-msg

```diff
@@ -38,6 +38,13 @@ instructions='#\
 #   PERF:   - performance improvement\
 #   STYLE:  - no logic impact (indentation, comments)\
 #   WIP:    - Work In Progress not ready for merge\
+#
+# The first line of the commit message should be 72 characters or less; the
```

Comment:
Maybe `should preferably be 72 characters...` (The word "preferably" makes it sound less strict, more friendly, to me).